client/ayon_core/hosts/nuke/plugins/publish/collect_headless_farm.py

```diff
@@ -0,0 +1,47 @@
+import pyblish.api
+
+
+class CollectHeadlessFarm(pyblish.api.ContextPlugin):
+    """Setup instances for headless farm submission."""
+
+    # Needs to be after CollectFromCreateContext
+    order = pyblish.api.CollectorOrder - 0.4
```

Comment:
Can it be even earlier?
```suggestion
    order = pyblish.api.CollectorOrder - 0.49
```